### DISCUSSION TITLE
New user help: screen stuck on creating codespace

### DISCUSSION THREAD

**Discussion by janyanas** [community/community#64265] [New to GitHub]
Hello. I am using my iPad to perform codespace however the window opens a new tab that says creating codespace without taking me to the space. I’m not sure if I am doing something wrong. I have tried using chrome and safari and the app but its not working

**Comment by queenofcorgis** [comment]
Hi there, 

Have you checked your WiFi's firewall settings?

**Comment by janyanas** [comment]
Ummm. No. How do I do that?

On Aug 21, 2023, at 8:53 AM, Lili ***@***.***> wrote:

﻿

Hi there,

Have you checked your WiFi's firewall settings?

**Comment by sithsiri** [comment]
Encountering same issue on iPad

**Comment by queenofcorgis** [comment]
Is your device fully up to date? Looks like that helped solve users who were experiencing the same in [this discussion](https://github.com/orgs/community/discussions/63015).

**Comment by janyanas** [comment]
Yes it is!

On Aug 23, 2023, at 9:59 AM, Lili ***@***.***> wrote:

﻿

Is your device fully up to date? Looks like that helped solve users who were experiencing the same in this discussion<https://github.com/orgs/community/discussions/63015>.
